crates/tx5/src/state/test.rs

```diff
@@ -105,12 +124,19 @@ impl Test {
     pub async fn shutdown(mut self) {
         self.shutdown = true;
 
-        let enc = prometheus::TextEncoder::new();
-        let mut buf = Vec::new();
-        use prometheus::Encoder;
-        enc.encode(&prometheus::default_registry().gather(), &mut buf)
+        let result = self
+            .influxive
+            .query(
+                r#"from(bucket: "influxive")
+    |> range(start: -15m, stop: now())
+    "#,
+            )
+            .await
             .unwrap();
-        println!("{}", String::from_utf8_lossy(&buf));
+
+        println!("{result}");
```

Comment:
Is there anything that can be asserted on `result` to check this automatically?

Reply:
mmm... the state logic isn't really the right place for this. Since there's no real connection, no real metrics are actually collected. This was more about making sure there aren't any panics in the processing mechanisms.

I'll add a check for metrics in a higher level integration test.

Reply:
[done](https://github.com/holochain/tx5/pull/45/commits/105b9b631a545867722a28718d21205b39ee2c97) - unfortunately, due to the use of `opentelemetry_api::global`, I now have to run the tests single threaded... until I get around to implementing a test workaround for that.